azure-pipelines.yml

```diff
@@ -11,6 +11,15 @@ variables:
   buildConfiguration: 'release'
   buildPlatform: 'anycpu'
 
+resources:
+  repositories:
+  - repository: das-platform-building-blocks
+    type: github
+    name: SkillsFundingAgency/das-platform-building-blocks
+    ref: refs/heads/master
+    endpoint: "GitHub (SFA)"
+      
+
```

Comment:
I'm going to be super irritating, but could the carriage returns be shortened to one line? :(

Reply:
Fixed